test/models/partner_test.rb

```diff
@@ -103,11 +104,17 @@ class PartnerTest < ActiveSupport::TestCase
 
 class PartnerServiceAreaTest < ActiveSupport::TestCase
   setup do
-    @partner = create(:partner)
-    @neighbourhood = create(:neighbourhood)
+    @neighbourhood = neighbourhoods(:one)
+    @user = create(:user)
+    @user.neighbourhoods << @neighbourhood
+    @partner = build(:partner, address: nil, accessed_by_user: @user)
   end
 
   test 'is valid when empty' do
+    # give partner an address the user admins
```

Comment:
this comment accidentally a word and im not sure what it is supposed to be saying? hehe :sweat_smile: